src/components/pages/homepage/visualizations.js

```diff
@@ -8,32 +7,32 @@ import wsjGraph from '../../../images/homepage-visualizations/wsj.png'
 import visualizationsStyles from './visualizations.module.scss'
 
 const Visualizations = () => (
-  <div className={visualizationsStyles.visualizationsWrapper}>
+  <div className={`layout-region ${visualizationsStyles.visualizationsModule}`}>
     <Container>
-      <h2>From the visualization gallery</h2>
-      <Flex flexWrap="wrap" className={visualizationsStyles.visualization}>
+      <h2 className="hed-primary">From the visualization gallery</h2>
+      <ul className={visualizationsStyles.visualizationsList}>
         <Visualization
           image={nytGraph}
-          altText="New York Times chart"
           sourceLink="https://www.nytimes.com/interactive/2020/03/17/us/coronavirus-testing-data.html"
           sourceTitle="U.S. Lags in Coronavirus Testing After Slow Response to Outbreak"
           publicationName="The New York Times"
+          itemClass={visualizationsStyles.visualization}
```

Comment:
You should be able to import the module.scss file into the visualization.js file so you don't have to pass styles as props.

Or we can just put the `<Visualization> component into this file... it's not used anywhere else but here.

Reply:
@kevee Oh got it, that’s good feedback—thanks, Kevin. Does aeb1ba6fba2e7d02c391c181c200c6155cc7915e look better?